### DISCUSSION TITLE
PowerShell Script for OneDrive

### DISCUSSION THREAD

**Discussion by Jaisanj** [community/community#129308] [Programming Help]
### Select Topic Area

General

### Body

Hello Everyone,
I am new to Git Hub and learning the use GitHub. Apologies if I have posted at the wrong place.
I have created a folder in each user's OneDrive with the name 'K drive', and migrated data in that folder.
I need the size of that folder from each user's OneDrive in a file through Powershell.

Thank you

**Comment by LiteBrite82** [comment]
Hi @Jaisanj ,

Thanks for being a part of the GitHub Community, we're glad you're here!

It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Comment by ebndev** [comment]
Hi @Jaisanj, thanks for being a part of the GitHub Community and sharing this with us!

Boosting for visibility as you posted this last week. While we may not always have users who are able to help with your issue, I hope you continue to post and engage in Discussions 😄
